queue_services/entity-filer/src/entity_filer/resources/worker.py

```diff
@@ -80,6 +80,7 @@
 )
 from entity_filer.filing_processors.filing_components import name_request
 from entity_filer.services import queue
+from entity_filer.services.business_service import BusinessService
```

Comment:
I think for now this is the correct approach (at least it's the one I also decided to take), which is to copy the business service instead of importing from legal api.